web/web.go

```diff
@@ -146,9 +146,13 @@ func New(st local.Storage, qe *promql.Engine, rm *rules.Manager, status *Prometh
 	instrf := prometheus.InstrumentHandlerFunc
 	instrh := prometheus.InstrumentHandler
 
-	router.Get("/", instrf("status", h.status))
+	router.Get("/", instrf("graph", h.graph))
```

Comment:
That's why I'm a fan of the input box solution. It makes it useful for the common case of looking up expressions, but still gives us space to put in status information and in particular warning/error boxes for when the prometheus is falling over which will help with user support.